constants/networks.ts

```diff
@@ -43,3 +43,6 @@ export const nameToIdMapping: { [name: string]: NetworkId } = {
 	optimism: 10,
 	'optimism-kovan': 69,
 };
+
+export const productionNetworks = ['mainnet', 'optimism'];
```

Comment:
I don't love adding new arrays for this. Ideally we could put this in the mapping above. 

optimism: {
  id: 10,
  isMainnet: true
}

Reply:
Adressed